### ISSUE TITLE
Whitespace Breaks Equality In CG Describe

### ISSUE THREAD

**Issue by feocco** [Dabz/kafka-streams-migration#1]
The eventual output of `kafka-consumer-groups` needs to be trimmed to ensure we don't get stuck in a loop. Example:
```
Wait for Kafka Streams lag to reach 0
.......................................................
```

This can occur with the following lines:
* https://github.com/Dabz/kafka-streams-migration/blob/051645a0e309c9660194ea9ed0da59763c1a2b4c/migrate#L15
* https://github.com/Dabz/kafka-streams-migration/blob/051645a0e309c9660194ea9ed0da59763c1a2b4c/migrate#L23

To resolve this, I piped the `wc` output into `sed -e 's/^[[:space:]]*//'`. Example:
```
until [[ `docker-compose exec broker kafka-consumer-groups --describe --bootstrap-server localhost:19092 --group streams-wordcount  | grep 'streams-wordcount-' | awk '{ print $6 }' | grep -v '0' | grep -v '1' | wc -l | sed -e 's/^[[:space:]]*//'` == '0' ]]; do
  echo -n .
  sleep 5
done
```

**Comment by Dabz** [comment]
Good point, I updated that in https://github.com/Dabz/kafka-streams-migration/commit/903bb22f99ae6c6286d03e32f386ae2cf48f93e5 ;)

**Comment by Dabz** [comment]
And it's also fixed in #2, I merged the PR. Let me close this issue ;)
